rskj-core/src/main/java/org/ethereum/db/ReceiptStore.java

```diff
@@ -34,7 +35,7 @@
 
     TransactionInfo get(byte[] transactionHash);
 
-    TransactionInfo get(byte[] transactionHash, byte[] blockHash, BlockStore store);
+    Optional<TransactionInfo> get(byte[] transactionHash, byte[] blockHash);
```

Comment:
I think we may add a comment somewhere in this class stating the design decision of returning `Optional` instead of `null` so maybe a future refactor could change the other two methods for keeping the interface consistent

Reply:
I think it's so obvious...